Dz2.py

```diff
@@ -0,0 +1,10 @@
+i = (input("Введите текст: "))
+if (i.isdigit()) == True:
+    I = int(i)
+    if I % 2 == 0:
+        print("Вы ввели парное число")
+    else:
+        print("Вы ввели не парное число")
+else:
+    x = (len(i))
+    print("Вы ввели слово длинною в " + str(x) + " символа(ов)")
```

Comment:
You can use f-string to put some value inside of a string. In this case you can avoid type changing and can simply use a variable, like this:
```
phrase = 'I love Python'
count = 2
print(f'My favorite phrase is {phrase}, I can repeat it {count} times')